tests/publiccloud/ssh_interactive_start.pm

```diff
@@ -19,27 +19,14 @@ sub run {
 
     $self->{provider} = $args->{my_provider};    # required for cleanup
 
-    # Only initialize tunnels, if not previously done
-    if (!get_var('_SSH_TUNNELS_INITIALIZED', 0)) {
-        # This ensures that we have used the setup console, even if no module was run before.
-        my $setup_console = current_console();
-
-        # Establish the tunnel (it will stay active in foreground and occupy this console!)
-        select_console('tunnel-console');
-        ssh_interactive_tunnel($args->{my_instance});
-
-        # Enable ssh connection on setup console, this is done normally with the
-        # first activation hook in susedistribution:activate_console()
-        if ($setup_console !~ /tunnel/) {
-            select_console($setup_console);
-            # The verbose output is visible only at the tunnel-console -
-            #   it doesn't interfere with tests as it isn't piped to /dev/sshserial
-            script_run('ssh -E /var/tmp/ssh_sut.log -vt sut', timeout => 0);
-        }
-    }
+    # Initialize ssh tunnel for the serial device, if not yet happened
+    ssh_interactive_tunnel($args->{my_instance}) if (get_var('_SSH_TUNNELS_INITIALIZED', 0) == 0);
     die("expect ssh serial") unless (get_var('SERIALDEV') =~ /ssh/);
+    # The serial terminal needs to be activated manually, as it requires the $self argument
+    $self->select_serial_terminal();
+    enter_cmd('ssh -t sut');
```

Comment:
The `select_serial_terminal` subroutine does not required `$self`.

Reply:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/opensusebasetest.pm#L1295

I might not require the argument right now, but the function signature is meant to receive `$self` and that's also how it is used in most test cases.

I might not be needed in the current implementation, but relying on that assumption is dangerous.

Reply:
OK. But it would be worth improvement in separate pull request.

Reply:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/15725